pkg/chassis/rpc.go

```diff
@@ -11,6 +15,13 @@ const (
 	Grpc
 )
 
+const (
+	grpcKeepaliveTime        = 30 * time.Second
```

Comment:
Just a note that these will need to be configurable in the future

Reply:
Totally agree!

Reply:
Should we make backlog tickets for this sort of thing?

Reply:
That's a good idea